contracts/modules/STO/Capped/CappedSTOStorage.sol

```diff
@@ -5,6 +5,7 @@ pragma solidity ^0.5.0;
  */
 contract CappedSTOStorage {
 
+    bytes32 constant OPERATOR = "OPERATOR";
```

Comment:
please verify if this is a duplicate declaration. `OPERATOR` and `ADMIN` are already defined in `ModuleStorage` inherited via `STO -> Module -> ModuleStorage`